js/views/userPageVw.js

```diff
@@ -595,6 +607,11 @@ module.exports = Backbone.View.extend({
 
   storeClick: function(e){
     "use strict";
+    var select = document.getElementById('categories');
+    if (select.value != "all"){
+        select.value = "all"
+        this.categoryChanged();
```

Comment:
Do you need to call this.categoryChanged() here? I think this will fire the change event that's already being listed to.